java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,22 @@
+package com.zoolatech.lecture1.tasks._1;
```

Comment:
Please remove files from the PR that are not related to the second homework and were already defined here: https://github.com/zoolacode/zoola-academy-homework/pull/91/files